docs/en/development/style.md

```diff
@@ -2,816 +2,831 @@
 
 ## General recommendations
 
-1. The following are recommendations, not requirements.
-2. If you are editing code, it makes sense to follow the formatting of the existing code.
-3. Code style is needed for consistency. Consistency makes it easier to read the code, and it also makes it easier to search the code.
-4. Many of the rules do not have logical reasons; they are dictated by established practices.
+**1.** The following are recommendations, not requirements.
 
-## Formatting
+**2.** If you are editing code, it makes sense to follow the formatting of the existing code.
+
+**3.** Code style is needed for consistency. Consistency makes it easier to read the code, and it also makes it easier to search the code.
 
-1. Most of the formatting will be done automatically by `clang-format`.
+**4.** Many of the rules do not have logical reasons; they are dictated by established practices.
 
-1. Offsets are 4 spaces. Configure your development environment so that a tab adds four spaces.
+## Formatting
 
-1. A left curly bracket must be separated on a new line. (And the right one, as well.)
+**1.** Most of the formatting will be done automatically by `clang-format`.
 
-   ```cpp
-    inline void readBoolText(bool & x, ReadBuffer & buf)
-    {
-        char tmp = '0';
-        readChar(tmp, buf);
-        x = tmp != '0';
-    }
-   ```
+**2.** Offsets are 4 spaces. Configure your development environment so that a tab adds four spaces.
```

Comment:
Bad translation. It should be "indentation", not "offsets".

Reply:
I'll fix it